low-level/message_handlers/service_msg_handler.py

```diff
@@ -91,8 +91,13 @@ def initialize(self, conf_reader, msgQlist, product):
         self.node_id = Conf.get(GLOBAL_CONF, NODE_ID_KEY, "SN01")
         self.cluster_id = Conf.get(GLOBAL_CONF, CLUSTER_ID_KEY, "CC01")
         self.storage_set_id = Conf.get(GLOBAL_CONF, STORAGE_SET_ID_KEY, "ST01")
-        self.monitored_services = Conf.get(SSPL_CONF,
-                                    f'{SERVICEMONITOR}>{MONITORED_SERVICES}')
+        services = Conf.get(SSPL_CONF, f'{SERVICEMONITOR}>{MONITORED_SERVICES}')
+        node_type = Conf.get(GLOBAL_CONF, NODE_TYPE_KEY, "VM").lower()
+        excluded_services = {
+            "vm": {"multipathd.service"},
+            "hw": {}
+        }
+        self.monitored_services = list(set(services) - excluded_services[node_type])

```

Comment:
Should this change be made in Node health as well?
In node health also, the list of external services is coming from:
`services = Conf.get(SSPL_CONF, f'{SERVICEMONITOR}>{MONITORED_SERVICES}')`

Reply:
Good finding. I will address that separately.

Reply:
Okay